gpt4all-bindings/typescript/prompt.cc

```diff
@@ -5,10 +5,10 @@ TsfnContext::TsfnContext(Napi::Env env, const PromptWorkContext& pc)
     : deferred_(Napi::Promise::Deferred::New(env)), pc(pc) {
 }
 
-std::mutex mtx;
-static thread_local std::string res;
+static std::string *res;
```

Comment:
As said on the Discord, you might want to consider using the following instead:
```cpp
namespace {
    std::string *res;
}
```
Otherwise, the `res` name is externally visible.

Reply:
Oh yeah, thanks for the reminder.